index.js

```diff
@@ -1,30 +1,36 @@
 'use strict'
 
-const holidays = require('./holidays/2017')
+const holidays = require('./holidays/')
+
+const holiday = obj => {
```

Comment:
If I run:
```js
holiday()
```

it returns me this error:
```bash
/Users/buuh7921/github/holiday/index.js:7
  const {year = date.getFullYear(), month, day, country = 'us'} = obj
                                                                  ^
TypeError: Cannot match against 'undefined' or 'null'.
```

You probably want to use it like this:
```js
const holiday = (opts = {}) => {
  ...
}
```

Ps: change `obj` to `opts`